mandatory/2-function-creation.js

```diff
@@ -15,26 +21,58 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if(totalPrice > 200){
+    totalPrice -= totalPrice * 0.1;
+    return totalPrice;
+  }
+  if(totalPrice < 200){
+    totalPrice -= totalPrice * 0.5;
+    return totalPrice;
+  }
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
```

Comment:
This one is almost there - but there's a couple of small problems.
- In the code below you're doing `number % 2 == 1;` - did you want to use an `if` statement here?
- Right now, this code will result in an infinite loop (this means the while loop will never end). I would suggest you use the VsCode debugger to step through the code line by line. Check what's happening to the `number` variable while you are doing this. (let us know if you need help with that 😄)